src/features/HelpAndKnowledge/SolutionsHelp/SolutionDetails/PointsOfContact/index.tsx

```diff
@@ -21,7 +22,22 @@ export const PointsOfContact = ({
 
   return (
     <div>
-      <h2 className="margin-top-0">{t('navigation.points-of-contact')}</h2>
+      <h2 className="margin-top-0 margin-bottom-2">
+        {t('mailboxesAndTeamMembers')}
+      </h2>
+      <div className="height-3 margin-bottom-2">
+        <Button type="button" className="usa-button usa-button--unstyled">
```

Comment:
You don't need to add these classes I believe as they should already be built into Truss' components.  Truss uses USWDS style guide and utility classes.  Also I think the Button component has an optional prop for `unstyled`

Reply:
it does! I was wondering about that. Thank you!